python/example/network_ring.py

```diff
@@ -121,9 +121,8 @@ def global_properties(self, kind):
 ncells = 4
 recipe = ring_recipe(ncells)
 
-# (12) Create an execution context using all locally available threads and simulation
-ctx = arbor.context(threads="avail_threads")
-sim = arbor.simulation(recipe, ctx)
+# (12) Create a simulation using all locally available threads
```

Comment:
Yeah, it's short, yet less clear what's going on. 
```suggestion
# (12) Create a simulation using the default settings:
# - Use all threads available
# - Use round-robin distribution of cells across groups with one cell per group
# - No GPU
# - No MPI
# - Empty label dictionary
# Other constructors of simulation can be used to change all of these.
```

Reply:
Good point. Slightly modified in upcoming commit. Label dict left out, it's set elsewhere.